### ISSUE TITLE
useActiveElement not working

### ISSUE THREAD

**Issue by gkubisa** [vueuse/vueuse#3774]
### Describe the bug

The ref returned by `useActiveElement` does not trigger reactivity, if it is used only within a `computed`.

It can be reproduced using the example from the `useActiveElement` docs.
1. Open https://stackblitz.com/edit/vitejs-vite-d3gdsx/?file=package.json,src%2FApp.vue
2. Move focus between the input fields

Expected: The "key" is updated to indicate which element is selected.
Actual: The "key" is not updated.

Interestingly, when I tried reproducing the same within my app, the reactivity worked until I moved focus to the browser address bar and than back to the app.

Here's are the core lines extracted from the reproduction above.

```vue
<script setup>
const activeElement = useActiveElement();
// Adding the watch below makes reactivity work but it should not be necessary.
// watch(activeElement, () => {}, { flush: 'sync' });
const key = computed(() => activeElement.value?.dataset?.id || 'null');
</script>
<template>
<span>{{ key }}</span>
</template>
```

The issue was previously reported in https://github.com/vueuse/vueuse/issues/3760, however, it was closed without being fixed.

### Reproduction

https://stackblitz.com/edit/vitejs-vite-d3gdsx/?file=package.json,src%2FApp.vue

### System Info

```Shell
System:
    OS: Linux 6.5 Ubuntu 22.04.3 LTS 22.04.3 LTS (Jammy Jellyfish)
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz
    Memory: 5.77 GB / 15.33 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 18.18.2 - ~/.nvm/versions/node/v18.18.2/bin/node
    npm: 9.8.1 - ~/.nvm/versions/node/v18.18.2/bin/npm
    pnpm: 8.15.1 - ~/.local/share/pnpm/pnpm
  Browsers:
    Chrome: 121.0.6167.85
  npmPackages:
    @vueuse/components: 10.7.2 => 10.7.2 
    @vueuse/core: 10.7.2 => 10.7.2 
    @vueuse/integrations: 10.7.2 => 10.7.2 
    @vueuse/math: 10.7.2 => 10.7.2 
    vue: 3.4.15 => 3.4.15
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by bayareawebpro** [comment]
I just noticed the demo in the docs isn't working either:  https://vueuse.org/core/useActiveElement/

**Comment by mitjans** [comment]
After investigating a little bit, the problem seems to be related to using the value of `computedWithControl` inside a `computed`:

```typescript
const a = computedWithControl(...)
const b = computed(() => /* if `a` is used here, a warning is thrown in Vue 3.4.19 and `b` will stop working at some point */)
```

**Comment by meteorlxy** [comment]
@mitjans The warning your mentioned might be the same thing I reported in #3794. I created a new issue for that because it seems not related to `useActiveElement`

**Comment by imvimol** [comment]
@JarvisH can you help me here https://github.com/vueuse/vueuse/issues/3799

**Comment by jaketig** [comment]
Should be fixed now via #3815 
Demos are working again!
